doc/requirements.txt

```diff
@@ -16,5 +16,5 @@ sphinx-gallery==0.5.0
 sphinxcontrib-bibtex==0.4.2
 tensorflow-tensorboard==0.1.8
 tensorflow==2.0.1
-thewalrus>=0.11
+http://xanadu-wheels.s3-website-us-east-1.amazonaws.com/thewalrus-0.13.0.dev0+20200527003218-cp36-cp36m-manylinux2010_x86_64.whl
```

Comment:
How did it go with `doc/requirements.txt`?

Reply:
it works!